packages/fuels/Cargo.toml

```
@@ -45,3 +45,4 @@ std = [
 # TODO: To be removed once https://github.com/FuelLabs/fuels-rs/issues/881 is unblocked.
 test-type-paths = []
 fuel-core-lib = ["fuels-test-helpers?/fuel-core-lib", "dep:fuel-core"]
+rocksdb = ["fuels-test-helpers?/rocksdb"]
```

```suggestion
rocksdb = ["fuels-test-helpers?/rocksdb"]

```

GitHub likes its new lines